### ISSUE TITLE
Rebase privcount-tor to tor 0.3.0.6

### ISSUE THREAD

**Issue by teor2345** [privcount/privcount#242]
My exit needs this fix:
```
Stop rejecting all IPv6 traffic on Exits whose exit policy rejects any IPv6 addresses...
```

**Comment by teor2345** [comment]
We should probably use the latest stable version, which is now 0.3.0.6.

**Comment by teor2345** [comment]
This is done in:
https://github.com/privcount/tor/tree/privcount-0.2.9.10
https://github.com/privcount/tor/tree/privcount-0.3.0.6

But is awaiting a review on:
https://github.com/privcount/tor/pull/5

Before it is pushed to the privcount branch itself.

**Comment by teor2345** [comment]
Fixed by pull privcount/tor#5, I'll push the main branch after testing.
